src/dataflow/DataflowVisitor.java

```diff
@@ -17,4 +20,13 @@ public DataflowVisitor(DataflowChecker checker, InferenceChecker ichecker,
             BaseAnnotatedTypeFactory factory, boolean infer) {
         super(checker, ichecker, factory, infer);
     }
+
+    /**
+     * Skip this test because every class has at least one non-top upper bound
+     * determined by the Java type it extends.
+     */
```

Comment:
constructor always produces objects of that type
TODO: validate constructor result appropriately